### PR TITLE
optionally return old or out of order NACKed packets 

### PR DESCRIPTION
**Pull request by mariat-atg**

### CODE REVIEW

File: lib/Room.js [hunk 1]
```diff
@@ -270,10 +270,30 @@ class Room extends EnhancedEventEmitter
...
+								logger.debug('"createRtpStreamer()" has ffmpeg-proxy rtcp-fb: %s', dbgfeed);
+
+							let rtcpFeedback = codec.rtcpFeedback.filter((val) => val.type !== 'ffmpeg-proxy');
+							codec.rtcpFeedback = rtcpFeedback;
+						}
+					});
+				}
+				else
+				{
+					logger.debug('"createRtpStreamer()" no changes in ffmpeg-proxy rtcp-fb '); // TODO: remove before checkin
```
Comment (jseeley78) [inline lib/Room.js]: don't forget to remove?

**Review by jseeley78 — APPROVED** [review]
Change seems reasonable, hope it improves the concealment errors!

File: lib/Room.js (hunk 1 above)
Comment (mariat-atg) [reply lib/Room.js]: Cleaned up yesterday after testing, thanks!

File: lib/Room.js [hunk 2]
```diff
@@ -270,10 +270,22 @@ class Room extends EnhancedEventEmitter
 				};
 
 				// TODO: The app should provide his own RTP capabilities.
-				const consumerRtpParameters =
+				let consumerRtpParameters =
```
Comment (artushin) [inline lib/Room.js:273]: Technically, this is still a const. The ref to the consumerRtpParameters object never changes. You're just modifying it's internal parameters below

**Review by artushin — APPROVED** [review]

**Pull request merged**